alias.go

```diff
@@ -32,10 +32,13 @@ var (
 	// match the existing environ variable to minimize surprises
 	WithDisableWarnings = option.WithNullAlerter
 	WithDisableTools    = option.WithDisableTools
+	WithPathOverrides   = option.WithPathOverrides
 )
 
 type SnapshotOptions = option.SnapshotOptions
 
+type PathOverridesOptions = option.PathOverrides
```

Comment:
Let's call the alias `PathOverrides` instead of `PathOverridesOptions` in order to match the existing alias scheme.

Reply:
sure thing!